streamwebs_frontend/streamwebs/templates/streamwebs/base.html

```diff
@@ -1,19 +1,21 @@
 {% load staticfiles %}
-{% load pipeline %}
-
 <!DOCTYPE html>
 <html lang="en">
 <head>
     <meta charset="utf-8" />
     <title>{% block title %}Streamwebs{% endblock %}</title>
     <link rel="stylesheet" type="text/css" href="{% static 'streamwebs/style.css' %}" />
     <link rel="shortcut icon" href="{% static 'streamwebs/images/favicon.ico' %}" />
-    {% javascript 'libraries' %}
+    {% load pipeline %}
```

Comment:
You need to keep the `{% javascript %}` tag here. `{% load pipeline %}` just tells Pipeline to find the static files and collect them, `{% javascript 'libraries' %} causes it to actually inject the <script> tags.